packages/Telephony/Actions/TimeoutChoiceInput.cs

```diff
@@ -64,11 +64,14 @@ public override async Task<DialogTurnResult> ContinueDialogAsync(DialogContext d
             if (!interrupted && activity.Type != ActivityTypes.Message && activity.Name == ActivityEventNames.ContinueConversation)
             {
                 //Set max turns so that we evaluate the default when we visit the inputdialog.
-                MaxTurnCount = 1;
+                if (MaxTurnCount != null)
+                {
+                    dc.State.SetValue(TURN_COUNT_PROPERTY, this.MaxTurnCount.GetValue(dc.State) + 1);
+                }
```

Comment:
Should this comment be deleted since the functionality and comment were incorrect?

Reply:
No this comment is still relevant. We are just setting max turns for the action in a different way now